src/optimizer/loadOptimizers.ts

```diff
@@ -23,30 +20,17 @@ export async function loadOptimizers(
     relativePath = `./${relativePath}`;
   }
 
-  // read the plugins folder, filter to only .ts/.js files, and map to unique module paths (without extensions)
-  const optimizerModules = uniq(
-    fs
-      .readdirSync(folder, { withFileTypes: true })
-      .filter(f => f.isFile() && /^[^.]+.*\.(t|j)s$/.test(f.name))
-      .map(f => `${relativePath}/${f.name.match(/^([^.]+)/)[1]}`)
-  );
+  // Import optimizers from the specified folder
+  const Optimizers: { property: OptimizerPlugin } = await import(relativePath);
```

Comment:
I haven't fully reviewed the changes here and I don't fully understand this, but I figured I'd post it in case anyone knew before I looked too much into it. But when I `npm pack` this branch and use it in FSH Online, this line gives me an error: 

```
Uncaught (in promise) Error: Cannot find module './plugins'
```

It appears that it doesn't like the variable passed into `import`. If this was a string path passed directly into `import`, it seems to work just fine. I'm not exactly sure why or what the right change is, but figured I'd at least post this for now!

Reply:
Commenting here for documentation purposes. As [noted in webpack's documentation](https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import), performing imports with a variable containing the target path are not allowed. You __can__ perform imports from a dynamic string though, which is the workaround that @jafeltra and I settled on!